Cargo.toml

```diff
@@ -6,3 +6,13 @@ edition = "2018"
 
 [dependencies]
 libc = "*"
+
+[dev-dependencies]
+lang_tester = "0.3"
+tempdir = "*"
```

Comment:
Is there a reason we're not using a specific version of this library?

Reply:
Yes, laziness ;) I've set this to `0.3.7`

Reply:
We should generally use just the major and minors so `0.3` would be better (cargo treats the patch level as a minimum requirement, so it's almost never worth specifying IMHO).

Reply:
I see